corgi/tasks/rhel_compose.py

```diff
@@ -63,16 +63,15 @@ def get_builds(
         relations_query = relations_query.filter(product_ref=stream_name)
 
     processed_builds = 0
-    for build_id, build_type in (
-        relations_query.values_list("build_id", "build_type").distinct().iterator()
+    for build_id in (
+        relations_query.filter(software_build=None)
+        .exclude(build_type=SoftwareBuild.Type.CENTOS)
```

Comment:
This is good, but let's add a comment explaining why:
```suggestion
        # We don't support modular builds in composes for CENTOS
        .exclude(build_type=SoftwareBuild.Type.CENTOS)
```
We should also move up the .filter() and .exclude() logic to where relations_query is originally defined at the start of this task on line 57, just to keep things cleaner.

Reply:
Ok, good idea.